test/src/sdk-harness/test_projects/storage_bytes/mod.rs

```diff
@@ -0,0 +1,190 @@
+));
+
+async fn test_storage_bytes_instance() -> TestStorageBytesContract {
+    let wallet = launch_provider_and_get_wallet().await;
+    let id = Contract::deploy(
+        "test_projects/storage_bytes/out/debug/storage_bytes.bin",
+        &wallet,
+        TxParameters::default(),
+        StorageConfiguration::with_storage_path(Some(
+            "test_projects/storage_bytes/out/debug/storage_bytes-storage_slots.json".to_string(),
+        )),
+    )
+    .await
+    .unwrap();
+
+    TestStorageBytesContract::new(id.clone(), wallet)
+}
+
+#[tokio::test]
+async fn returns_empty_bytes() {
+    let instance = test_storage_bytes_instance().await;
+
+    let input = vec![];
+
+    assert_eq!(instance.methods().len().call().await.unwrap().value, 0);
+
+    let _ = instance
+        .methods()
+        .assert_stored_bytes(input.clone())
+        .call()
+        .await;
+}
+
+#[tokio::test]
+async fn stores_byte() {
+    let instance = test_storage_bytes_instance().await;
+
+    let input = vec![1u8];
+
+    assert_eq!(instance.methods().len().call().await.unwrap().value, 0);
+
+    let _ = instance.methods().store_bytes(input.clone()).call().await;
+
+    assert_eq!(instance.methods().len().call().await.unwrap().value, 1);
+
+    let _ = instance
+        .methods()
+        .assert_stored_bytes(input.clone())
+        .call()
+        .await;
+}
+
+#[tokio::test]
+async fn stores_8_bytes() {
+    let instance = test_storage_bytes_instance().await;
+
+    let input = vec![1u8, 2u8, 3u8, 4u8, 5u8, 6u8, 7u8, 8u8];
+
+    assert_eq!(instance.methods().len().call().await.unwrap().value, 0);
+
+    let _ = instance.methods().store_bytes(input.clone()).call().await;
+
+    assert_eq!(instance.methods().len().call().await.unwrap().value, 8);
+
+    let _ = instance.methods().assert_stored_bytes(input).call().await;
+}
+
+#[tokio::test]
+async fn stores_32_bytes() {
+    let instance = test_storage_bytes_instance().await;
+
+    let input = vec![
+        1u8, 2u8, 3u8, 4u8, 5u8, 6u8, 7u8, 8u8, 1u8, 2u8, 3u8, 4u8, 5u8, 6u8, 7u8, 8u8, 1u8, 2u8,
+        3u8, 4u8, 5u8, 6u8, 7u8, 8u8, 1u8, 2u8, 3u8, 4u8, 5u8, 6u8, 7u8, 8u8,
+    ];
+
+    assert_eq!(instance.methods().len().call().await.unwrap().value, 0);
+
+    let _ = instance.methods().store_bytes(input.clone()).call().await;
+
+    assert_eq!(instance.methods().len().call().await.unwrap().value, 32);
+
+    let _ = instance.methods().assert_stored_bytes(input).call().await;
+}
+
+#[tokio::test]
+async fn stores_string_as_bytes() {
+    let instance = test_storage_bytes_instance().await;
+
+    let input = String::from("Fuel is blazingly fast!");
+
+    assert_eq!(instance.methods().len().call().await.unwrap().value, 0);
+
+    let _ = instance
+        .methods()
+        .store_bytes(input.clone().as_bytes().into())
+        .call()
+        .await;
+
+    assert_eq!(
+        instance.methods().len().call().await.unwrap().value,
+        input.clone().as_bytes().len() as u64
```

Comment:
Why `as u64`?

Reply:
This defaults to a `usize`

Reply:
You cannot compare a `u64` and a `usize`